layersvt/device_simulation.cpp

```diff
@@ -20,14 +20,8 @@
 
 /*
  * layersvt/device_simulation.cpp - The VK_LAYER_LUNARG_device_simulation layer.
- * This DevSim layer simulates a device by loading a configuration file to override values that would normally be returned from a
- * Vulkan implementation.
- *
- * NOTICE: This is a "version 0" prerelease of the DevSim layer.
- * Behaviors may change before a general "version 1.x" public release, particularly the configuration fileformat.
- * vkjson-style JSON was used as a configuration fileformat for this initial DevSim release because it was available; however, it is
- * not really suited to the DevSim usemodel.
- * Recommend refining the format (and a schema for validation) specifically for use by DevSim.
+ * This DevSim layer simulates a device by loading a JSON configuration file to override values that would normally be returned
+ * from a Vulkan implementation.  The configuration files must validate with the DevSim schema; see kSchemaDevsim100.
```

Comment:
Hard to know what "kSchemaDevsim100" is at this point in the file.  Maybe say:

See the kSchemaDevsim variable below to find the URI of the schema.